flang/test/Lower/intrinsic-procedures.f90

```diff
@@ -169,6 +169,17 @@ function btest_test(i, j)
   btest_test = btest(i, j)
 end
 
+! CPU_TIME
+! CHECK-LABEL: cpu_time_test
+subroutine cpu_time_test
+  ! CHECK: %[[t:[0-9]+]] = fir.alloca f32 {bindc_name = "t"
+  real :: t
```

Comment:
I suggest making 't' a dummy argument so there's no chance that the store to 't' gets optimized away.